docs/section16/Rule16-7.md

```diff
@@ -0,0 +1,33 @@
+# Elevators – Rule 16-7  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-7  
+**Rule Description:** When included in the proposed design, the baseline elevator cab lighting power density shall be 3.14 W/ft2  
+**Rule Assertion:** B-RMD = Expected Value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16  
+**Data Lookup:** None  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+N/A
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- For each elevator in the baseline RMD: `for elevator in B_RMD...elevators`
+  - Get the elevator cab lighting power: `cab_lighting_power_b = elevator.cab_lighting_power`
+  - Get the elevator cab area: `cab_area_b = elevator.cab_area`
```

Comment:
I know we have discussed this a million times, just want to verify that it will be clear to the RCT development team that you are expecting ft^2 since the schema is in metric.

Reply:
Added units to rule assertion so that they don't get lost in translation. Comparison will consider the units.